profile_template

```diff
@@ -22,11 +22,19 @@ minimum_jsonapi_version_explanation:
 # ask questions/start discussions about your extension.
 discussion_url: http://tets.com
 
-author_name: Ethan Resnick
-author_email: ethan.resnick@gmail.com
+# For multiple editors, use a pipe (|) and place each editor on a new line. Use
```

Comment:
If possible, I think it would be nicer here to use a yaml array of objects; i.e., something like:

```yaml
editors:
  - name: Jane Doe
    email: janedoe@gmail.com
  - name: John Doe
    email: johndoe@gmail.com
    website: example.com
```

That way, people don't have to match up the `n/a`s, and a comment can just indicate that name + email are required for each author.

I say "If possible" because I seem to recall that this structure was really hard to work with in liquid for some reason... but I don't see now why it would be.

Reply:
@ethanresnick, if you can, would you mind making this change? I'm not at all familiar with how the project is setup/built etc.

I think you have access make changes to this PR.